zpbase/functions.php

```diff
@@ -311,7 +311,7 @@ function printBaseSlideShow() {
 		} else {
 			if (isset($_POST['favorites_page'])) {
 				$albumobj = $_myFavorites;
-				$returnpath = rewrite_path($_myFavorites->getLink() . '/' . $pagenumber, FULLWEBPATH . '/index.php?p=favorites' . '&page=' . $pagenumber);
+				$returnpath = getCustomPageURL('favorites', 'page=' . $pagenumber);
```

Comment:
Is this the favorites page you @acrylian refer to? I have no clue how to change that.

Reply:
Yes, exactly. Actually just `$returnpath = $_myFavorites->getLink($pagenumber); `; should be enough here (see: https://docs.zenphoto.org/1.5.3/source-class-AlbumBase.html#566-585)